test/v3/createPool/weighted.integration.test.ts

```diff
@@ -0,0 +1,75 @@
+// pnpm test -- createPool/weighted.integration.test.ts
+
+import {
+    Address,
+    createTestClient,
+    http,
+    parseEther,
+    publicActions,
+    walletActions,
+    zeroAddress,
+} from 'viem';
+import { CHAINS, ChainId, PoolType, TokenType } from '../../../src';
```

Comment:
You should import directly from `src` for consistency and to make sure everything is properly exported.